experiments/functional/backup_and_restore/minio-volume-snapshot-location.yml

```diff
@@ -10,5 +10,6 @@ spec:
       bucket: velero
       provider: aws
       region: minio
+      #profile: new-profile
```

Comment:
We need to deploy two volume snapshot location, one without `spec.config.profile`(which will be considered as `default`) and one with `spec.config.profile`(with value=`new-profile`).

Reply:
We are doing regex operation on volumesnapshotlocation.yml to uncomment `#profile:`